vtds_cluster_kvm/private/config/virtual_node_template.xml

```diff
@@ -56,13 +56,13 @@
     <emulator>/usr/bin/qemu-system-x86_64</emulator>
```

Comment:
REVIEWERS: this XML template is filled out using Jinja2 for use when deploying qcow live image based Virtual Node configurations. It is the XML that is fed to `virsh define` to define the VM that implements the Virtual Node. Here I have changed the name of the parameter from `file_name` to `file_path` to make it clearer what is needed here.